dist/Interfaces/user/UserSubscription.d.ts

```diff
@@ -0,0 +1,51 @@
+import User from './User';
+import { Subscription } from '../subscription/Subscription';
+import { SubscriptionPricingPlan } from '../subscription/SubscriptionPricingPlan';
+import { SubscriptionTier } from '../subscription/SubscriptionTier';
+import { SubscriptionAddon } from '../subscription/SubscriptionAddon';
+export declare type UserSubscriptionReasonForEndIdType = 1 | 2 | 3 | 4 | 5 | 6 | 7;
```

Comment:
Do we want to define the number types here? What if other statuses are added for example?